doc/guides/debugger.md

```diff
@@ -38,7 +38,7 @@ To confirm mrdb was installed properly, run mrdb with the `--version` option:
 
 ```bash
 $ mrdb --version
-mruby 2.0.0 (2018-12-11)
+mruby 2.0.1 (2019-4-4)
```

Comment:
Output should be `mruby 2.0.1 (2019-04-04)` if #4353 is included, I think.

Reply:
Indeed. But 2.0.1 was forked before #4353

Reply:
Ah, I didn't notice forked. Thank you.